terraform/image-proxy/main.tf

```
@@ -0,0 +1,20 @@
+variable "domain" { type = string }
+variable "sitename" { type = string }
+variable "conveyor_address" { type = string }
```

This should probably not be named this?

```suggestion
variable "backend_address" { type = string }
```

Maybe?

---

on it 🫡